bindings/pydeck/Makefile

```diff
@@ -0,0 +1,45 @@
+init:
+	cd ../..
+	yarn bootstrap
+	cd -
+	pip install -r requirements.txt
+	pip install -r requirements-dev.txt
+	pip install -e .
+	make prepare-jupyter
+
+prepare-jupyter:
+	jupyter nbextension install --sys-prefix --symlink --overwrite --py pydeck
+	jupyter nbextension enable --sys-prefix --py pydeck
+	jupyter labextension install @jupyter-widgets/jupyterlab-manager
+	cd ../../modules/jupyter-widget/
+	jupyter labextension install .
+
+test:
+	pytest
+
+docs:
+	cd docs
+	make html
+
+# e.g. `make RELEASE_TYPE=MAJOR bump-version
+bump-version:
+	python bump_version.py $RELEASE_TYPE
+	export $PYDECK_VERSION=$@
```

Comment:
Does this work?  What does it do?  I would think your trying to do `export PYDECK_VERSION=1.2.3`, but `$@` should equal `bump-version`.  And your expanding the `PYDECK_VERSION` variable not assigning to it aren't you?